assets/components/productPage/productPageContentBlock/productPageContentBlock.scss

```diff
@@ -18,11 +18,33 @@
     @include multiline(4, $border);
   }
   /*
-  div > $selector adds some leeway to use react elements with extra divs 
-  until preact.fragment gets here
+  All our ContentBlocks have the behaviour of rendering 
+  with a border between them if there's two with the same 
+  background colour in a row.
+
+  These selectors attempt to enforce this by applying the 
+  border to .selector + .selector (adjacent sibling) 
+  
+  The second selector applies the rule to the first child 
+  of an element wrapped in an extra div if it matches the 
+  classname (because React makes you do that sometimes)
+
+  I hope this explains it:
+  
+  +----------------+
+  | blue block     |
+  +----------------+
+  +----------------+
+  | wrapping div   |
+  |+--------------+|
+  || another blue ||
+  || block        ||
+  |+--------------+|
+  +----------------+
+
   */
   & + &,
-  & + div > &  {
+  & + div > &:first-child  {
```

Comment:
There was a bug here before where the selector would have added a border to all children of the same color instead of just the first one, this is the correct behaviour.